### PR TITLE
Store and retrieve data from PODs

### PR DESCRIPTION
**Pull request by Davidmc07**
* Convert typescript objects to solid datasets using an Assembler class
* Use SPARQL queries to get data from PODs
* Convert the query results to typescript objects using the Assembler
* Add the necesary calls to the PODManager methods to use the PODs data

### CODE REVIEW

**Review by uo282892 — APPROVED** [review]

**Review by UO283428 — APPROVED** [review]

**Review by UO283069 — APPROVED** [review]

File: webapp/src/adapters/solid/PODManager.ts [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+     */
+    public async getAllMaps(): Promise<Array<Map>> {
+        let path:string = this.getBaseUrl() + '/data/maps/';
+
+        let urls = await this.getContainedUrls(path);
+        let maps = await this.getMapPreviews(urls);
+        return maps;
+    }
+
+    public async getPlace(url:string): Promise<Place> {
```
Comment (Pelayo-Reguera) [inline webapp/src/adapters/solid/PODManager.ts:55]: A place also have a category and images

File: webapp/src/adapters/solid/PODManager.ts
Comment (Pelayo-Reguera) [inline webapp/src/adapters/solid/PODManager.ts:1]: Everything related to PlaceMarks I think that are not stored in PODs

File: webapp/src/domain/Place.tsx [hunk 2]
```diff
@@ -2,29 +2,26 @@
...
       this._photos = photos;
+      this._uuid = (id === undefined) ? crypto.randomUUID() : id;
     }
 
-    get uuid(): string | undefined {
+    get uuid(): string {
         return this._uuid;
     }
 
-    set uuid(name: string | undefined) {
```
Comment (Pelayo-Reguera) [inline webapp/src/domain/Place.tsx:24]: Although generating the UUID here it's OK I think the setter property shouldn't be deleted as, if I'm not wrong, it should be recovered when the place is got from the POD

**Review by Pelayo-Reguera — CHANGES_REQUESTED** [review]

File: webapp/src/domain/Place.tsx (hunk 2 above)
Comment (Davidmc07) [reply webapp/src/domain/Place.tsx:24]: Yes, that's why there is an optional id parameter in the constructor. After the data is recovered from the POD, the uuid is passed as an argument when creating the object.

Comment (Pelayo-Reguera) [reply webapp/src/domain/Place.tsx:24]: Oh OK, I didn’t see it

File: webapp/src/adapters/solid/PODManager.ts (hunk 1 above)
Comment (Davidmc07) [reply webapp/src/adapters/solid/PODManager.ts:55]: It didn't when i created the branch and that issue is assigned to someone else. I just store and retrieve the existing entities. When that part is implemented i will update the POD manager.

**Review by Pelayo-Reguera — APPROVED** [review]

**Pull request merged**